pymatgen/io/phonopy.py

```diff
@@ -196,6 +189,33 @@ def get_ph_bs_symm_line_from_dict(bands_dict, has_nac=False, labels_dict=None):
     return ph_bs
 
 
+def get_eig_b(b, q, structure):
+    """
+
```

Comment:
This docstring needs to be completed before merging.

Reply:
Decided to remove this function again and include this in the main code. At a certain stage of the code development, I needed this function more than once but this is now obsolete.